examples/reactnative/rn063example/README.md

```diff
@@ -31,6 +31,12 @@ This project was bootstrapped with `react-native init`.
 
     The API key can be found in the Bugsnag settings for your project.
 
+1. For iOS only, install the project dependencies using: 
```

Comment:
My preference would be to follow the docs and suggest they `cd` into the directory and run `pod install`. It's a bit less magic and more understandable.

Unless @bengourley feels otherwise, in which case we should update the docs.

Reply:
The reason why I didn't was because the next line (I believe) relies on being in the directory above - so it would need two extra statements to be added.  Happy to change it if we're happy with that though.

Reply:
That seems like overkill and I think would be obvious to a RN developer. How about making it less explicit?

```suggestion
1. For iOS only, install the project dependencies from the `ios/` directory using: 
```

Reply:
Wording changed as suggested and option removed from the command.